PFERD/crawl/ilias/ilias_web_crawler.py

```diff
@@ -23,10 +23,20 @@
 from .ilias_html_cleaner import clean, insert_base_markup
 from .kit_ilias_html import (IliasElementType, IliasForumThread, IliasLearningModulePage, IliasPage,
                              IliasPageElement, _sanitize_path_name, parse_ilias_forum_export)
+from .shibboleth_login import ShibbolethLogin
 
 TargetType = Union[str, int]
 
 
+class ShibbolethLoginType():
+    pass
+
+
+class LocalLoginType():
```

Comment:
```suggestion
class LoginTypeLocal:
```
Might be easier to search for, if there are ever more than one. I guess this can also be a `@dataclass`?